src/styles/index.scss

```diff
@@ -173,6 +186,53 @@ font-size: 20px;
 	letter-spacing: 0.2px;
 	font-size: 17px;
 }
-.view-btn:hover{
-	background: #097205;;
+.view-btn:hover {
+	background: #097205;
+}
+
+.form-wrapper {
+  @media (max-width: 768px) {
```

Comment:
Put this in the initial `.form-wrapper` definition

Reply:
what do yo  mean here